tests/behaviour/query/language/fetch.rs

```diff
@@ -0,0 +1,23 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+#![allow(unexpected_cfgs, reason = "features defined in Bazel targets aren't currently communicated to Cargo")]
+
+use steps::Context;
+
+#[tokio::test]
+async fn test() {
+    // Bazel specific path: when running the test in bazel, the external data from
+    // @vaticle_typedb_behaviour is stored in a directory that is a sibling to
+    // the working directory.
+    #[cfg(feature = "bazel")]
+    let path = "../vaticle_typedb_behaviour/query/language/fetch.feature";
+
+    #[cfg(not(feature = "bazel"))]
+    let path = "bazel-typedb/external/vaticle_typedb_behaviour/query/language/fetch.feature";
```

Comment:
Make sure you rebase and use the new format after Dmitrii's merge - this won't be right anymore

Reply:
Sure, I was waiting for it